spec/models/proceeding_spec.rb

```diff
@@ -32,6 +32,14 @@
                               :used_delegated_functions_reported_on)
   }
 
+  it { is_expected.to have_one(:attempts_to_settle).dependent(:destroy) }
+  it { is_expected.to have_one(:chances_of_success).dependent(:destroy) }
+  it { is_expected.to have_one(:vary_order).dependent(:destroy) }
+
+  it { is_expected.to have_many(:proceeding_linked_children).dependent(:destroy) }
+  it { is_expected.to have_many(:involved_children).through(:proceeding_linked_children).source(:involved_child) }
+  it { is_expected.to have_many(:scope_limitations).dependent(:destroy) }
```

Comment:
happy to get rid of these if we do not think they are worth it.  I find them helpful in a TDD flow

Reply:
cool, will remove.